app/src/main/java/uz/direction/noteapp/ShowNote.kt

```diff
@@ -0,0 +1,19 @@
+package uz.direction.noteapp
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import uz.direction.noteapp.databinding.ActivityMainBinding
+import uz.direction.noteapp.databinding.ActivityShowNoteBinding
+
+class ShowNote : AppCompatActivity() {
```

Comment:
Not the best name for activity. First of all, class name shouldn't be verb, the second always try to add suffix activity as it will be easier to identify that it is activity

Reply:
Okay